example/src/main/java/org/xmtp/android/example/ClientManager.kt

```diff
@@ -32,9 +31,8 @@ object ClientManager {
                 appVersion = "XMTPAndroidExample/v1.0.0",
                 isSecure = true
             ),
-            enableV3 = true,
             appContext = appContext,
-            dbEncryptionKey = encryptionKey
+            dbEncryptionKey = encryptionKey!!
```

Comment:
Looks like this is effectively an `.unwrap()`? What would you think if we changed the above assignment on lines 22-26? Claude helped me out here. Ignore if you want.

```
val encryptionKey = keyUtil.retrieveKey(address)?.takeUnless { it.isEmpty() }
        ?: SecureRandom().generateSeed(32).also { keyUtil.storeKey(address, it) }
```

Reply:
Thats some witchy Kotlin right there! 😄 Love it.